LeetCodeSolutions/src/main/java/N-Queens.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+public class NQueens 
```

Comment:
Please make sure the build passes for your PR, there's just one thing left and that is the name of the file and the public class are not same

Reply:
Done, @Pradyuman7 please check!